automan/automan_website/settings.py

```diff
@@ -112,12 +113,15 @@
 # Database
 # https://docs.djangoproject.com/en/1.11/ref/settings/#databases
 # MySQL
+mysql_access_dict = secret.get_mysql_username_password()
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.mysql',
         'NAME': os.environ.get("MYSQL_DB_NAME"),
-        'USER': os.environ.get("MYSQL_USER"),
-        'PASSWORD': os.environ.get("MYSQL_PASSWORD"),
+        # 'USER': os.environ.get("MYSQL_USER"),
+        # 'PASSWORD': os.environ.get("MYSQL_PASSWORD"),
```

Comment:
It's okay to have this comment out turned off.

Reply:
thanks. got it.

Reply:
so quickly! thanks!!